<!DOCTYPE html>
<html lang="en">
   <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
      <meta name="description" content="" />
      <meta name="author" content="" />
      <title>Ecosystem Genomics Institute</title>
      <!-- Favicon-->
      <link rel="icon" type="image/x-icon" href="assets/favicon.ico" />
      <!-- Bootstrap Icons-->
      <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css" rel="stylesheet" />
      <!-- Google fonts-->
      <link href="https://fonts.googleapis.com/css?family=Merriweather+Sans:400,700" rel="stylesheet" />
      <link href="https://fonts.googleapis.com/css?family=Merriweather:400,300,300italic,400italic,700,700italic" rel="stylesheet" type="text/css" />
      <!-- SimpleLightbox plugin CSS-->
      <link href="https://cdnjs.cloudflare.com/ajax/libs/SimpleLightbox/2.1.0/simpleLightbox.min.css" rel="stylesheet" />
      <!-- Core theme CSS (includes Bootstrap)-->
      <link href="css/egi.css" rel="stylesheet" />
      <link href="css/styles.css" rel="stylesheet" />
   </head>
   <body id="page-top">
      <!-- Navigation-->
      <nav class="navbar navbar-expand-lg navbar-light fixed-top py-3" id="mainNav">
         <div class="container px-4 px-lg-5">
            <a class="navbar-brand" href="#page-top">
            <img src="assets/img/egilogo-01.png" alt="" height="60">
            </a>
            <button class="navbar-toggler navbar-toggler-right" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarResponsive">
               <ul class="navbar-nav ms-auto my-2 my-lg-0">
                  <li class="nav-item"><a class="nav-link" href="#about">About</a></li>
                  <li class="nav-item"><a class="nav-link" href="#team">People & Partners</a></li>
                  <li class="nav-item"><a class="nav-link" href="#seminars">Seminars</a></li>
                  <li class="nav-item"><a class="nav-link" href="#funding">Projects</a></li>
                  <li class="nav-item"><a class="nav-link" href="#GIDP">Graduate Programs</a></li>
                  <li class="nav-item"><a class="nav-link" href="https://www.youtube.com/user/Bio5institute"><i class="bi bi-youtube"></i></a></li>
               </ul>
            </div>
         </div>
      </nav>
      <!-- Masthead-->
      <header class="masthead">
         <div class="container h-100">
            <div class="row gx-4 gx-lg-5 h-100 align-items-center justify-content-center text-center">
               <div class="col-lg-8 align-self-end">
                  <h1 class="text-white font-weight-bold mt-3">Ecosystem Genomics Initiative</h1>
                  <hr class="divider" />
               </div>
               <div class="col-lg-8 align-self-baseline">
                  <p class="text-white-75 mb-5">Ecosystem Genomics seeks to address the grand challenge of scaling biology from genes to ecosystems, simultaneously advancing theory and practical solutions to problems ranging from global change to human health. The Arizona Ecosystem Genomics Initiative (AEGI) is a nexus for coalescing University of Arizona strengths in environmental science, ecology, genome-enabled science, and ‘big data’ cyberinfrastucture to address this grand challenge.</p>
                  <a class="btn btn-primary btn-xl" href="#about">Find Out More</a>
               </div>
            </div>
         </div>
      </header>
      <!-- About-->
      <section class="page-section bg-primary" id="about">
         <div class="container">
            <div class="row justify-content-center">
               <div class="col-lg-8 text-center">
                  <h2 class="text-white mt-3">What is the Ecosystem Genomics Initiative?</h2>
                  <hr class="divider divider-light" />
                  <p class="text-white-75 mb-4">In 2012 University of Arizona (UA) envisioned a grand opportunity for implementing a new science of ‘Ecosystem Genomics’. The Ecosystem Genomics Initiative seeks to “Discover the emergent properties and processes of ecosystems through ‘top down’ analysis of communities, populations and organisms, and ‘bottom up’ analysis of genomes, transcriptomes and metabolomes.”</p>
                  <div class="accordion accordion-flush" id="accordionFlushExample">
                     <h2 class="text-white mt-0 mb-3">What is Ecosystem Genomics?</h2>
                     <p>
                        <button class="btn btn-primary" type="button" data-bs-toggle="collapse" data-bs-target="#collapseExample" aria-expanded="false" aria-controls="collapseExample">
                        Read More
                        </button>
                     </p>
                     <div class="text-white-75 collapse" id="collapseExample">
                        <div>
                           <p>The term ‘ecosystem genomics’ has been used in the past by both geneticists and ecosystem biologists as an aspiration for disciplinary integration; yet, the aim of placing genomes into a framework capable of predicting the rates and outcomes of ecosystem processes has remained elusive. The Ecosystem Genomics Initiative will provide a basis to investigate principles for integrating data and theory to synthesize a unified, genes-to-ecosystems framework for predicting ecosystem processes and managing them to sustainable ends. The discipline of genome biology is poised to drive broad, integrative research projects in ecosystem ecology. For example, the Earth Microbiome Project aims to ‘analyze microbial communities across the globe and within the environmental parameter space of different biomes’. The NIH launched the Human Microbiome Project to understand how the ‘ecosystem’ of the human body controls the expression of complex diseases. The Biology Directorate at NSF established a community Wiki to relieve bottlenecks that impede progress on the genome-to-phenome grand challenge. At the same time, ecosystem scientists have shown that some of the complex feedbacks in Earth Systems Models (ESMs) are more accurately represented when genome and trait data are included. Ecosystem biologists have asked: ‘how can we use genomic details about organisms and communities to accurately predict the roles of specific traits in controlling ecosystem processes and their relation to challenges such as climate change’?</p>
                           <p>The central factors that are missing, but required, to translate genomics into ecosystem processes are foundational theory, case studies and models. The absence of these elements prevents us from addressing questions such as: Why are some genes more important than others in determining the aggregate traits of ecosystems? To what degree do plant genomes control the community assembly of soil microbial genomes, and produce a microbiome that determines ecosystem aggregate traits? What are the scaling ‘conveniences’ that will enable us to represent the translation from genes to traits to processes in models of the entire earth system? To link genomics to processes, and thus create the predictive framework we seek, foundational theory must be developed to define the complete genes-to-ecosystems translation. Case studies should be used to align measurements of genes, traits and processes in complementary environments and systems. Through this alignment, we can be sure we have all the elements of the theory represented in specific ecosystems. With theory and case studies in hand, we can iteratively develop, test, and refine hierarchically-organized models. The discipline of ecosystem genomics that we envision will advance a broad framework for predictive ecosystem genomics. Our research will build this framework through an integrated portfolio of complementary questions, scales, and ecosystems. Through research and education activities we will create international impact by designing new types of models that inform global climate policy, identifying genes and genomic interactions that enhance crop yield, and training a new generation of students to join the national workforce in fields such as ecosystem management, medical genetics and food security
                           </p>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
         </div>
      </section>
      <!-- Services-->
      <section class="page-section" id="services">
         <div class="container">
            <h2 class="text-center mt-5">How does UA benefit from such an initiative?</h2>
            <div class="row">
               <div class="col-lg-3 col-md-6 text-center">
                  <div class="mt-5">
                     <div class="mb-2"><i class="bi-gem fs-1 text-primary"></i></div>
                     <p class="text-muted mb-0">Funding opportunities are diverse and generous including government and private foundations.</p>
                  </div>
               </div>
               <div class="col-lg-3 col-md-6 text-center">
                  <div class="mt-5">
                     <div class="mb-2"><i class="bi-laptop fs-1 text-primary"></i></div>
                     <p class="text-muted mb-0">The science is inclusive and embraces strengths across campus (from Biosphere 2 to CyVerse ).</p>
                  </div>
               </div>
               <div class="col-lg-3 col-md-6 text-center">
                  <div class="mt-5">
                     <div class="mb-2"><i class="bi-globe fs-1 text-primary"></i></div>
                     <p class="text-muted mb-0">The Ecosystem Genomics Graduate Interdisciplinary Program (ECGN GIDP) nucleates intellectual exchange in this area and prepares UA graduates in an emerging field.</p>
                  </div>
               </div>
               <div class="col-lg-3 col-md-6 text-center">
                  <div class="mt-5">
                     <div class="mb-2"><i class="bi-phone fs-1 text-primary"></i></div>
                     <p class="text-muted mb-0">“BIG DATA” needs interface with flagship NSF programs ($434M NEON), and UA-led iPlant Collaborative (uniting initiatives in Health Informatics, iPlant and microbiology via iMicrobe).</p>
                  </div>
               </div>
            </div>
         </div>
      </section>
      <!-- Call to action-->
      <section class="page-section bg-primary text-white" style="padding: 2rem">
         <div class="container px-1 px-lg-1 text-center">
            <h2 class="mb-1 mt-5">Who is Involved</h2>
            <hr class="divider" />
         </div>
      <!-- Portfolio-->
      <div class="page-section bg-primary" id="team">
         <div class="container-fluid">
             <div class="row">
                 <div class="col-12">
                    <ul class="nav nav-tabs" role="tablist">
                        <li class="nav-item">
                          <a class="nav-link active" data-bs-toggle="tab" href="#current">Current Faculty</a>
                        </li>
                        <li class="nav-item">
                          <a class="nav-link text-white" data-bs-toggle="tab" href="#past">Past Faculty</a>
                        </li>
                      </ul>
                 </div>
             </div>

             <!-- Tab panes -->
  <div class="tab-content">
    <div id="current" class="container tab-pane active"><br>
        <div class="row">
            <div class="col-md-12 text-center">
               <div class="row mt-3">
                  <div class="col-md-3">
                     <img src="assets/img/team/Barberan.jpg" class="img-thumbnail team-img" alt="..."></a>
                     <p class="team-name">Albert Barberan <a href="https://barberanalbert.wixsite.com/labpage"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseBarberan" role="button" aria-expanded="false" aria-controls="collapseBarberan">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/Betsy-Arnold.jpg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">Betsy Arnold <a href="www.arnoldlab.net"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseArnold" role="button" aria-expanded="false" aria-controls="collapseArnold">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/hurwitz.png" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">Bonnie Hurwitz <a href="http://hurwitzlab.org/"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseHurwitz" role="button" aria-expanded="false" aria-controls="collapseHurwitz">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/moore.jpeg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">David Moore <a href="https://environment.arizona.edu/person/david-j-moore"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseMoore" role="button" aria-expanded="false" aria-controls="collapseMoore">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div id="bioGrp">
                     <div class="collapse" id="collapseBarberan" data-bs-parent="#bioGrp">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white">Albert Barberan's research focuses on bridging the gap between the disciplines of general ecology and 
                                 microbial ecology by analyzing large and complex molecular datasets. A more complete understanding of microbial
                                 processes and patterns is essential in order to grasp the relationships among ecosystem functions, global change, 
                                 and management of natural and working landscapes. Using molecular techniques and data analysis, The Barberan Lab explores the potential 
                                 for dust-associated microorganisms to successfully colonize distant environments, the genomic basis of microbial adaptation
                                 in soils, and the effect of invasive plants on soil microbial communities.
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseArnold" data-bs-parent="#bioGrp">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> Betsy is an evolutionary ecologist with expertise in plant microbiomes in both wild and agricultural ecosystems. 
                                 She and her group use the tools of microbiology, molecular ecology, informatics, systematics, organismal biology, and genomics to study the
                                 distributions and impacts of plant-associated microbes worldwide, with study sites that range from the Arctic to tropical rainforests. Her 
                                 interests in fungal biodiversity, plant-microbe interactions, fungal-bacterial dynamics, and molecular ecology are readily translated to animal 
                                 systems in diverse settings. She collaborates with agricultural experts in academia and industry, natural products chemists interested in pharmaceutical drug
                                 discovery, and academic partners interested in identifying and translating principles of biodiversity research to human endeavors. As Curator of 
                                 the Robert L. Gilbertson Mycological Herbarium she oversees the largest collection of fungi in the southwestern US. Betsy is a committed mentor, 
                                 an award-winning instructor, and is active in outreach, especially with high school students, K-12 teachers, and the broader Arizona community.
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseHurwitz" data-bs-parent="#bioGrp">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> Dr. Bonnie Hurwitz is an Assistant Professor of Agricultural and Biosystems Engineering at the University of Arizona and Bio5 Research Institute Fellow. She has worked as a computational biologist for nearly two decades on interdisciplinary projects in both industry and academia. Her research on the earth microbiome incorporates large-scale –omics datasets, high-throughput computing, and big data analytics to answer questions in systems biology. In particular, Dr. Hurwitz is interested how viruses re-engineer host metabolism and the implications on host-driven biogeochemical cycles in the ocean at a global scale. Dr. Hurwitz is well-cited for her work in computational biology in diverse areas from plant genomics to viral metagenomics with over 1300 citations.   
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseMoore" data-bs-parent="#bioGrp">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                                 Dave Moore uses ecological knowledge and quantitative models to improve mechanistic understanding of the uptake, storage and release of carbon from ecosystems to provide accurate predictions of the role of terrestrial ecosystems in global biogeochemical cycles.
                              </p>
                           </div>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
         </div>
         <div class="row">
            <div class="col-md-12 text-center">
               <div class="row mt-3">

                  <div class="col-md-3">
                     <img src="assets/img/team/jana.jpeg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">Jana U’Ren</p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseUren" role="button" aria-expanded="false" aria-controls="collapseUren"  data-parent="#bioGrp">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/Meredith.jpg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">Laura Meredith <a href="http://www.laurameredith.com/"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseMeredith" role="button" aria-expanded="false" aria-controls="collapseMeredith"  data-parent="#bioGrp">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/matzkin.png" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">Luciano Matzkin <a href="http://matzkinlab.org/"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseMatzkin" role="button" aria-expanded="false" aria-controls="collapseMatzkin"  data-parent="#bioGrp">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                    <img src="assets/img/team/sullivan.jpg" class="img-thumbnail team-img" alt="...">
                    <p class="team-name"> Matthew Sullivan <a href="http://u.osu.edu/viruslab/"><i class="bi bi-link"></i></a></p>
                    </p>
                    <p>
                       <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseSullivan" role="button" aria-expanded="false" aria-controls="collapseSullivan">
                       Show Bio
                       </a>
                    </p>
                 </div>
                  <div id="bioGrp2">
                     <div class="collapse" id="collapseUren" data-bs-parent="#bioGrp2">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                                 I am an integrative microbial ecologist with training in microbial ecology, evolutionary biology, mycology, and genomics. My primary research interest is on plant-microbial interactions, specifically on asymptomatic endophytic fungi. Endophytes occur within all plant species, where they can provide defense against herbivores and pathogens, as well as tolerance to heat and drought. Using field sampling, traditional culture-based methods, and next-generation sequencing of genomes and communities I am interested in characterizing the biotic and abiotic factors shaping the assembly of plant-associated fungal communities, how community structure and diversity impacts ecosystem function, and the evolutionary dynamics of fungal symbiont evolution in the context of closely related pathogens and saprotrophs.
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseMeredith" data-bs-parent="#bioGrp2">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white">
                                 My research team quantifies the microbial imprint on atmospheric composition and climate using an interdisciplinary set of methods, ranging from genomics to micrometeorology. We focus on resolving the genomic underpinnings of microbe-mediated biogeochemical transformations in soils that drive significant atmospheric fluxes. The goal of my work is to determine when and how projections of biogeochemical transformations are improved by better representation of underlying biological drivers.
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseMatzkin" data-bs-parent="#bioGrp2">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                                 The Matzkin lab investigates how the ecology of a species shapes patterns of variation at multiple levels (genes, pathways, transcriptomes, genomes, physiology, behavior and life history), how populations adapt to environmental shifts, the implication of ecological adaptation in the process of speciation and the role of sexual selection and sexual conflict in the evolution of reproductive incompatibilities. Our research revolves around these fundamental aspects of evolutionary biology. We have traditionally focused on a group of cactophilic Drosophila that inhabit the deserts of North America. These Drosophila species are an excellent system to study given that their ecology is well understood and the fact that we can perform many genetic, genomic, manipulative and life history experiments. Additionally, we work on understanding the genomic basis of resistance to Bacillus thuringiensis (Bt) toxins in Lepidopteran agricultural pest, to provide insight into the efficiency and long-term efficacy of insect pest control strategies.
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseSullivan" data-bs-parent="#bioGrp2">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                                 As the world faces global change and resource limitation, understanding the planet’s microbes (e.g. bacteria, archaea) becomes a necessity. This is because microbes drive the biogeochemistry that runs the planet, and are central to human endeavors, from food to health to industry. Viruses that infect microbes profoundly shape microbial populations and processes by acting as both major predators and sources of new genes. We work to understand viral impacts on globally important microbial processes. As empiricists, we test hypotheses through direct systems-level studies of natural populations, complemented by developing and studying model phage-host systems in the lab with the goal of generating the data required for predictive ecosystem modeling. Practically, we develop new ways to “see” viruses — in the microscope, in environmental sequence datasets, in experiments — i.e., we are learning how to “count” across different data types. Given this quantitative toolkit, current research questions include:
                                 1. What is a viral population or species?
                                 2. How do viral assemblages change over space and time, and what drives these patterns?
                                 3. How do bottom-up processes impact top-down predators?
                              </p>
                           </div>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
         </div>
         <div class="row">
            <div class="col-md-12 text-center">
               <div class="row mt-3">
                  <div class="col-md-3">
                     <img src="assets/img/team/nirav.jpg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name"> Nirav Merchant <a href="http://www.bio5.org/about/scientists/nirav-c-merchant"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseMerchant" role="button" aria-expanded="false" aria-controls="collapseMerchant">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/carini.jpeg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name"> Paul Carini <a href="http://carinilab.com/"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseCarini" role="button" aria-expanded="false" aria-controls="collapseCarini">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/gallery.jpeg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">Rachel Gallery <a href="http://rachelgallery.arizona.edu/"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseGallery" role="button" aria-expanded="false" aria-controls="collapseGallery">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div class="col-md-3">
                     <img src="assets/img/team/maier.jpeg" class="img-thumbnail team-img" alt="...">
                     <p class="team-name">Raina Maier <a href="http://swes.cals.arizona.edu/maier_lab/"><i class="bi bi-link"></i></a> </p>
                     <p>
                        <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseMaier" role="button" aria-expanded="false" aria-controls="collapseMaier">
                        Show Bio
                        </a>
                     </p>
                  </div>
                  <div id="bioGrp3">
                     <div class="collapse" id="collapseMerchant" data-bs-parent="#bioGrp3">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                              <p class="text-left text-white">
                                 Over the last two decades my work has focused on developing computational platforms and enabling technologies, primarily directed towards improving research productivity and collaboration for interdisciplinary teams and virtual organizations.
                                 The key thrust areas for my work encompass life cycle management for:
                              </p>
                              <ol class="text-white">
                                 <li>High throughput and automated bio sample processing systems</li>
                                 <li>Highly scalable data and metadata management systems</li>
                                 <li>High throughput and high performance computing systems</li>
                              </ol>
                              <p class="text-white">
                                 My recent work has been directed towards supporting pervasive computing needs for mHealth (mobile health) initiatives and health interventions, with focus on developing study management platforms that leverage cloud based telephony, messaging and video in conjunction with wearable’s and sensors.
                                 Platforms and tools developed by team are actively utilized for:
                              </p>
                              <ol class="text-white">
                                 <li>Managing bio samples and data for clinically certified (CAP/CLIA) NGS pipelines</li>
                                 <li>Large scale genotyping (million+ samples) platforms with robotic automation</li>
                                 <li>National scale Cyberinfrastructure (iPlant) that facilitate global team of researchers to effectively manage their data, computation and collaborations using a cohesive computational platform</li>
                              </ol>
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseCarini" data-bs-parent="#bioGrp3">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                                 The Carini lab is broadly focused on understanding how microbes help make Earth habitable for humans. This view leads us to diverse questions in both terrestrial and aquatic ecosystems with the goal of understanding how microbial communities transform important nutrients, remove pollutants, affect soil fertility and influence aquatic productivity. By studying the growth of microbial cultures, their genome sequences and their environmental distributions, we design experiments that help uncover new and unusual biogeochemical cycles and provide hypothesis-based explanations for long standing geochemical observations. Integrated ecosystem genomics-based approaches such as these are invaluable to help us understand the roles of microbes on Earth. We are also interested in how exposure to cosmopolitan environmental microbes that live in soil, water and dust influences human health. Using a combination of high-throughput cultivation and laboratory study, we seek a mechanistic understanding of the factors that influence the distributions of these microbes in their natural environment and the specific ways that exposure benefits (or is detrimental) to human health.
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseGallery" data-bs-parent="#bioGrp3">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                                 My research group studies the ecology of soil microbes. Microbes define soil health, support ecosystem services, and maintain plant diversity. They are critical to ecosystem resiliency, especially in the context of climate change and the conservation challenges we currently face. Working across a range of ecosystems from lowland tropical forests, to high elevation conifer forests, to semi-arid grasslands, we use field-based experiments, microbiological techniques, and contemporary genetic tools to test the effects of plant-microbe interactions on plant and microbe community diversity, understand how environmental shifts will alter these interactions, and accurately predict the subsequent impacts on ecosystem function. For more information please visit: rachelgallery.arizona.edu
                              </p>
                              </p>
                           </div>
                        </div>
                     </div>
                     <div class="collapse" id="collapseMaier" data-bs-parent="#bioGrp3">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white"> 
                                 My lab studies the microbial ecology and recovery of disturbed and marginal terrestrial ecosystems that are characterized by oligotrophy or nutrient poor conditions. We seek to understand patterns in microbial diversity that correlate with the sustainability of vegetation in disturbed regions such as mining wastes and arid desert ecosystems. Both culture- and molecular/phylogenetic-based approaches are used including metagenomic analysis of ecosystem communities. A second focal area is the study of “eco-friendly” microbial surfactants (biosurfactants) including discovery of new biosurfactants, elucidating the microbial physiology associated with biosurfactant production as well as the role of biosurfactants in microbial survival. This work is being translated into potential biomedical and environmental applications, e.g., harvesting rare earth elements from industrial waste streams.
                              </p>
                           </div>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
            <div class="row">
               <div class="col-md-12 text-center">
                  <div class="row mt-3">
                     <div class="col-md-3">
                        <img src="assets/img/team/monson.png" class="img-thumbnail team-img" alt="...">
                        <p class="team-name">Russell Monson<a href="http://cals.arizona.edu/research/monson/"><i class="bi bi-link"></i></a> </p>
                        <p>
                           <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseMonson" role="button" aria-expanded="false" aria-controls="collapseMonson">
                           Show Bio
                           </a>
                        </p>
                     </div>
                     <div class="col-md-3">
                        <img src="assets/img/team/Scott-Saleska.png" class="img-thumbnail team-img" alt="...">
                        <p class="team-name"> Scott Saleska | <small> Co-Founder </small><a href="http://www.saleskalab.org/"><i class="bi bi-link"></i></a></p>
                        <p>
                           <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseSaleska" role="button" aria-expanded="false" aria-controls="collapseSaleska">
                           Show Bio
                           </a>
                        </p>
                     </div>
                     <div class="col-md-3">
                        <img src="assets/img/team/rich.jpg" class="img-thumbnail team-img" alt="...">
                        <p class="team-name"> Virginia Rich <a href="http://www.openwetware.org/wiki/SWES-MEL"><i class="bi bi-link"></i></a></p>
                        <p>
                           <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseRich" role="button" aria-expanded="false" aria-controls="collapseRich">
                           Show Bio
                           </a>
                        </p>
                     </div>
                     <div id="bioGrp4">
                        <div class="collapse" id="collapseMonson" data-bs-parent="#bioGrp4">
                           <div class="card bg-primary">
                              <div class="card-body">
                                 <p class="text-left text-white"> 
                                    I study how climate change in the Western US is influencing the carbon and water cycles in mountain forests. I also study how the emissions of organic compounds from forests across the globe influence atmospheric chemistry, particularly with regard to the the lifetime of compounds that influence climate. In the past, I have studied plant-microorganism interactions in the alpine ecosystem nitrogen cycle and I have studied the evolution of C4 photosynthesis in plants.
                                 </p>
                              </div>
                           </div>
                        </div>
                        <div class="collapse" id="collapseSaleska" data-bs-parent="#bioGrp4">
                           <div class="card bg-primary">
                              <div class="card-body">
                                 <p class="text-left text-white"> My research focuses on what might be called “biogeochemical ecology,” asking questions about how climate interacts with plant physiology, demography, and ecological processes to influence or control biogeochemical cycling from local to global scales. Just one example of the need for more complete understanding in this area is the lack of species interactions in modern global climate models, even though such interactions can be critically important in controlling ecosystem carbon cycling and hence, feedbacks to climate. Progress has been limited by the difficulty of bridging the gap between local-scale ecological interactions and broader biogeochemical processes. I use multidisciplinary approaches that combine classical techniques of field ecology and forestry with advanced technological methods (e.g., the micrometeorological eddy covariance method, isotopic techniques) and modeling to integrate biogeochemical processes to ecosystem scales.</p>
                              </div>
                           </div>
                        </div>
                        <div class="collapse" id="collapseRich" data-bs-parent="#bioGrp4">
                           <div class="card bg-primary">
                              <div class="card-body">
                                 <p class="text-left text-white">Rich helped co-found the EGI and remains an affiliate, while now based at the Ohio State University. Her team studies how microbiomes of complex natural systems interact with anthropogenic change. As a molecular microbial ecosystems ecologist, she engages in highly collaborative multidisciplinary research with biogeochemists and modelers, to develop predictive understanding of emergent ecosystem response to change. She co-founded and co-led the IsoGenie Project (https://isogenie.osu.edu) to apply this systems-approach to understanding microbial climate feedbacks from thawing permafrost landscapes from 2010-2022. Rich now co-leads the EMERGE Biology Integration Institute (https://emerge-bii.github.io) which examines how the reciprocal interactions among molecules and biomes give rise to Emergent Ecosystem Response to chanGE, while integrating research, training and outreach.</p>
                              </div>
                           </div>
                        </div>
                     </div>
                     <div class="tab-content mb-3" id="pills-tabContent">
                        <div class="bio-pane tab-pane fade" id="bio-15" role="tabpanel" aria-labelledby="pills-contact-tab">
                           <div class="card bg-primary">
                              <div class="card-body">
                                 <p class="text-left text-white">
                                    My lab studies how microbes respond to, and in turn help shape, environmental change. We are particularly interested in global change interactions with biogeochemical cycling. Our lens into microbial community composition and function is through molecular “meta-omics” tools, which we bring to robust interdisciplinary collaborations with biogoechemists and modelers to generate a systems-level understanding of ecosystems undergoing change. To learn about our specific research projects, please visit our lab website at http://openwetware.org/wiki/Main_Page
                                 </p>
                              </div>
                           </div>
                        </div>
                        <div class="bio-pane tab-pane fade" id="bio-16" role="tabpanel" aria-labelledby="pills-contact-tab">
                           <div class="card bg-primary">
                              <div class="card-body">
                                 <p class="text-left text-white">
                                    As the world faces global change and resource limitation, understanding the planet’s microbes (e.g. bacteria, archaea) becomes a necessity. This is because microbes drive the biogeochemistry that runs the planet, and are central to human endeavors, from food to health to industry. Viruses that infect microbes profoundly shape microbial populations and processes by acting as both major predators and sources of new genes.
                                    We work to understand viral impacts on globally important microbial processes. As empiricists, we test hypotheses through direct systems-level studies of natural populations, complemented by developing and studying model phage-host systems in the lab with the goal of generating the data required for predictive ecosystem modeling. Practically, we develop new ways to “see” viruses — in the microscope, in environmental sequence datasets, in experiments — i.e., we are learning how to “count” across different data types.
                                    Given this quantitative toolkit, current research questions include:
                                 </p>
                                 <ol class="text-white">
                                    <li>What is a viral population or species?</li>
                                    <li>How do viral assemblages change over space and time, and what drives these patterns?</li>
                                    <li>How do bottom-up processes impact top-down predators?</li>
                                 </ol>
                              </div>
                           </div>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
      </div>
    </div>
    <div id="past" class="container tab-pane fade"><br>

        <div class="row">
            <div class="col-md-12 text-center">
               <div class="row mt-3">
                  
                <div class="col-md-3">
                    <img src="assets/img/team/fares.jpeg" class="img-thumbnail team-img" alt="...">
                    <p class="team-name">Hanna (Johnny) Fares <a href="http://mcb2.arizona.edu/fares/"><i class="bi bi-link"></i></a> </p>
                    <p>
                       <a class="btn btn-primary" data-bs-toggle="collapse" href="#collapseFares" role="button" aria-expanded="false" aria-controls="collapseFares"  data-parent="#bioGrp">
                       Show Bio
                       </a>
                    </p>
                 </div>
                  <div id="bioGrp">
                    <div class="collapse" id="collapseFares" data-bs-parent="#bioGrp2">
                        <div class="card bg-primary">
                           <div class="card-body">
                              <p class="text-left text-white">
                                 Hanna (Johnny) Fares, Ph.D., is Professor and Associate Head in the Department of Molecular and Cellular Biology. Dr. Fares uses Caenorhabditis elegans and mammalian cells as models to decipher molecular mechanisms of lysosome formation and the molecular basis for symptoms in lysosomal storage disorders.
                              </p>
                           </div>
                        </div>
                     </div>
                    
                  </div>
               </div>
            </div>
        </div>
    
    </div>

</div>
            </div>
         </div>
      </div>
      </section>
      <!-- Funding -->
      <section class="page-section" id="funding">
         <div class="container px-4 px-lg-5">
            <h2 class="text-center mt-5">Projects</h2>
            <hr class="divider"/>
            <div class="row">
                <div class="col-lg-4 col-md-6 col-12">
                    <p>A.E. Arnold (CoPI). National Science Foundation. Collaborative Research: Macrosystems Research Award: Distributions of macrofungi: quantifying ecosystem and climate drivers of fungal reproduction. 2022-2025. </p>

                </div>
                <div class="col-lg-4 col-md-6 col-12">
                    <p>A.E. Arnold (CoPI). National Science Foundation, Systematics and Biodiversity Science: Leveraging historical collections and new surveys to characterize foundational shifts in vital symbioses in the threatened Arctic. 2021-2023.</p>
                    
                </div>
                <div class="col-lg-4 col-md-6 col-12">
                    <p>A.E. Arnold (CoPI). National Science Foundation. Collaborative Research: Growing Convergence Research: Growing a new science of landscape terraformation: the convergence of rock, fluids, and life to form complex ecosystems across scales. Senior Personnel and Executive Committee member. 2021-2026.</p>
                    
                </div>
                <div class="col-lg-4 col-md-6 col-12">
                   <p>S. Saleska (PI), A.E. Arnold (CoPI), R.A. Gallery (CoPI), B.L. Hurwitz (CoPI), J. Croissant (CoPI), A. Barberan (SP), K. Dlugosch (SP), R. Ferriere (SP), L.M. Matzkin (SP), L. Meredith (SP), W. D. Pauli (SP), Y. Song (SP), M. Tfaily (SP), J. U’Ren (SP) S. Vincent (SP), R.E. Wing (SP). National Science Foundation, “NRT-URoL: <a href="https://bridges.arizona.edu/">BRIDGES</a> - Building Resources for InterDisciplinary training in Genomic and Ecosystem Sciences.” DGE-2022055 9/2020-8/2025. </p>
                </div>
                <div class="col-lg-4 col-md-6 col-12">
                   <p>
                     Y. Carriere (PD), B. Tabashnik (CoPD), L. M. Matzkin (CoPD), and J. Fabrick (CoPD). USDA-NIFA-BRAG. “Genomic Resistance Risk Assessment for Vip3-Producing Transgenic Crops in Helicoverpa zea”. 9/1/2020 - 8/31/2023. 
                   </p>
                </div>
                <div class="col-lg-4 col-md-6 col-12">
                   <p>
                     L. M. Matzkin (PI), F. Diaz (CoPI) and C. Cultid-Medina (CoPI). University of Arizona and the Mexican Council for Science and Technology (CONACYT)-CAZMEX. “The role of climate change and biological interactions in modulating reproductive isolation in cactophilic Drosophila”. 6/1/2020 - 4/30/2022
                   </p>
                </div>
                <div class="col-lg-4 col-md-6 col-12">
                  <p>
                     B. Tabashnik (PD), L. M. Matzkin (CoPD), J. Fabrick (CoPD) and Y. Carriere (CoPD). USDA-NIFA. “Population genomics of Bt resistance in Helicoverpa zea”. 6/1/2020 - 5/31/2023. 
                  </p>
               </div>
               <div class="col-lg-4 col-md-6 col-12">
                  <p>
                     
B. Tabashnik (PI), Y. Carriere (CoPI) and L. M. Matzkin (CoPI). Genetics of Pest Resistance to Bt Toxin Cry1Ac. University of Arizona-Core Facilities Pilot Program. 11/15/2019 - 11/14/2020. 

                  </p>
               </div>
               <div class="col-lg-4 col-md-6 col-12">
                  <p>
                     L. Matzkin (PI).  University of Arizona, CALS Innovation Venture Investment Project. “Inducible toxic ejaculate in Drosophila suzukii: An innovative approach for insect population control.” 01/2020 – 6/2021. 
                  </p>
               </div>
               <div class="col-lg-4 col-md-6 col-12">
                  <p>National Science Foundation Biology Integration Institute Virginia Rich</p>
                  
              </div>
            </div>
      </section>
      <!-- Seminars-->
      <section class="page-section bg-primary" id="seminars">
         <div class="container px-4 px-lg-5">
            <h2 class="text-center mt-5 text-white">Ecosystem Genomics Seminar Series</h2>
            <hr class="divider" />
            <div class="row gx-4 gx-lg-5">
               <div class="accordion" id="accordionExample">
                  <div class="accordion-item">
                     <h2 class="accordion-header" id="headingCurr">                                                          
                        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseCurr" aria-expanded="false" aria-controls="collapseCurr">
                        Spring 2022
                        </button>
                     </h2>
                     <div id="collapseCurr" class="accordion-collapse collapse" aria-labelledby="headingCurr" data-bs-parent="#accordionExample">
                        <div class="accordion-body text-center">
                           <br>
                           <p class="accordion-header">Visit <a href="https://docs.google.com/spreadsheets/d/1SkfRYedCLxo7OHrVjBHVhRkEU78ibfjcx3gpGMTLRl8/edit#gid=0">this link</a> for Spring 2022 speaker list and recordings</p>
                           <br>
                           <iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%237986CB&ctz=America%2FPhoenix&mode=AGENDA&showTabs=1&showCalendars=1&showTitle=0&src=MzEzdmk2cWhpNjMwb29zZG92bjU5bmw2NzRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%237986CB" style="border:solid 1px #777" width="800" height="300" frameborder="0" scrolling="no"></iframe>
                        </div>
                     </div>
                  </div>
                  <div class="accordion-item">
                     <h2 class="accordion-header" id="headingOne">                                                          
                        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
                        Fall 2019
                        </button>
                     </h2>
                     <div id="collapseOne" class="accordion-collapse collapse" aria-labelledby="headingOne" data-bs-parent="#accordionExample">
                        <div class="accordion-body">
                           <table width="800" class="table">
                              <tbody>
                                 <tr>
                                    <td width="125"><strong>Thursday</strong></td>
                                    <td width="125"><strong>Location</strong></td>
                                    <td width="150"><strong>Speaker</strong></td>
                                    <td width="400"><strong>Title</strong></td>
                                 </tr>
                                 <tr>
                                    <td width="125">August 29</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><b>Christiane Werner, </b>University of Freiburg</td>
                                    <td width="400"><em>Tracing plant volatile organic compound (VOC) emissions and CO<sub>2</sub> metabolism from sub-molecular to ecosystem scales by<sup>13</sup>C-labelling</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">September 5</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>David Baltrus</strong>, UA Plant Sciences</td>
                                    <td width="400"><em>How Biogeography Shapes Microbiome Structure </em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">September 12</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Lingling Shi</strong>, University of Göttingen</td>
                                    <td width="400"><em>Competition and complementarity of agroforestry components by carbon sequestration in soil: <sup>13</sup>C labeling and tracing</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">September 26</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Johnathan Williams</strong>, Max Planck Institute for Chemistry Mainz</td>
                                    <td width="400"><em>Measuring Volatile Organic Compounds from 325m over Amazon jungle </em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">October 3</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong><strong>Moira Hough (UA) &amp;&nbsp;</strong></strong><strong>Gemma Purser (CEH, Edinburgh) </strong></td>
                                    <td width="400">Hough: <em>Stable isotope tracing of litter through microbes and C-fluxes across an arctic permafrost thaw gradient</em>; Purser: <em>Bioenergy forests: A story about Eucalyptus and air quality in the UK. </em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">October 10</td>
                                    <td width="125"><strong>MRB 102*</strong></td>
                                    <td width="150"><strong>Jennifer Wisecaver</strong>, Purdue University</td>
                                    <td width="400"><em>Honor by association, leveraging co-expression networks for gene discovery in specialized metabolism (invited) </em>
                                 <tr>
                                    <td width="125">October 17</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><b>Nemiah Ladd</b>, University of Freiburg</td>
                                    <td width="400"><em>Hydrogen isotopes of lipid biomarkers: using multiple compounds to disentangle ecological and hydroclimate signals</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">October 24</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Jordan Krechmer</strong>, Aerodyne Research</td>
                                    <td width="400"><em>Developing Mass Spectrometry Instruments to Investigate Environmental Processes</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">October 31</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Jane Fudyma</strong>, UA Tfaily group &amp;<strong> Kathiravan Meeran</strong>, U Innsbruck, Bahn group</td>
                                    <td width="400"><em>Preliminary Soil Organic Matter Data Reveals Transition Zone and Spatial Differences within the Biosphere2 Tropical Rainforest Soil Profile</em> (Fudyma)&nbsp;<em>Effect of summer drought on the coupling of photosynthesis and soil respiration under current and future climate</em> (Meeran) </td>
                                 </tr>
                                 <tr>
                                    <td width="125">November 21</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><b>Marco Lehmann</b>, Swiss Federal Institute, WSL</td>
                                    <td width="400"><em>On the oxygen isotopic signature of plant assimilates</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">December 5</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong><b>Michaela Dippold</b>, University of Göttingen</strong></td>
                                    <td width="400"><em><em>Soil Fluxomics: Using position-specific labeling to unravel the metabolism of divers microbial communities in soils&nbsp;</em></em><em></em></td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </div>
                  </div>
                  <div class="accordion-item">
                     <h2 class="accordion-header" id="headingTwo">
                        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
                        Spring 2019
                        </button>
                     </h2>
                     <div id="collapseTwo" class="accordion-collapse collapse" aria-labelledby="headingTwo" data-bs-parent="#accordionExample">
                        <div class="accordion-body">
                           <table class="table" style="height: 1655px;" width="691">
                              <tbody>
                                 <tr>
                                    <td width="104"><strong>Thursday</strong></td>
                                    <td width="174"><strong>Location</strong></td>
                                    <td width="437"><strong>Speaker</strong></td>
                                    <td width="555"><strong>Title</strong></td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td></td>
                                    <td></td>
                                    <td></td>
                                 </tr>
                                 <tr>
                                    <td>Jan 17</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. Koenraad Van Doorslaer,UA, ACBS</td>
                                    <td width="555">Metagenomic insights into the evolution of viral oncogenes (<a href="https://drive.google.com/open?id=1xm-cEao_WHKFal-7MR3xpuk3jc78nsYh">Link to video recording)</a></td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Jan 24</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. William Burgos, Penn State, Department of Civil and Environmental Engineering</td>
                                    <td width="555">Acidic Pit Lakes of the Iberian Pyrite Belt, a Meta-Omics View</td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Feb 14</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. Bridget Barker, NAU</td>
                                    <td width="555">Detection and characterization of Onygenalean fungi associated with prairie dogs in northern Arizona</td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Feb 21</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. Betsy Arnold, UA, Plant Sciences</td>
                                    <td width="555">How fungal symbionts of leaves and seeds can influence the structure and function of plant communities (<a href="https://drive.google.com/open?id=1EAsLhLW5BsEKM0J7sxfm1Tc-LDEKek5Z">Link to video recording)</a></td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Feb 28</td>
                                    <td width="174">LTRR, Rm 110</td>
                                    <td>Dr. Liana Burghardt; Ecocsystem Genomics Faculty Candidate</td>
                                    <td width="555">From genes to communities: The legume-rhizobia nitrogen-fixing symbiosis</td>
                                 </tr>
                                 <tr>
                                    <td>Mar 14</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Postdocs: Dr. Alise Ponsero, BSE; Dr. Clairessa Brown, SWES</td>
                                    <td width="555">Ponsero: The promises and pitfalls of machine learning for detecting viruses in metagenomes. Brown: Understanding the ecological processes that influence microbial communities in arid soils&nbsp; (<a href="https://drive.google.com/open?id=1yXmjnCho3LL9BKC1TbeEG3gNpbcU7Eu-">Link to video recording)</a></td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Mar 28</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. Kerry Cooper, UA, ACBS</td>
                                    <td width="555">Food safety genomics: applying next generation sequencing to resolve pathogenesis and epidemiology of foodborne pathogens (<a href="https://drive.google.com/open?id=1ywRTlkVAfOtKwd6ImI5_lDFJAigp5f2I">Link to video recording)</a></td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Apr 4</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. Albert Barberán, UA, SWES</td>
                                    <td width="555">A bestiary of data visualization (<a href="https://arizona.zoom.us/j/764878406">Live Stream)</a></td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Apr 11</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. Luciano Matzkin, UA Entomology</td>
                                    <td width="555">Genomic, physiological and behavioral consequences of adaptation to local ecological conditions</td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Apr 18</td>
                                    <td width="174">Drachman Hall A120</td>
                                    <td width="437">Dr. Stephanie Dutkiewicz, Principal Research Scientist, MIT, Department of Earth, Atmospheric, and Planetary Sciences</td>
                                    <td width="555">Using ecosystem models to explore controls on marine phytoplankton diversity</td>
                                 </tr>
                                 <tr>
                                    <td></td>
                                    <td width="174"></td>
                                    <td width="437"></td>
                                    <td width="555"></td>
                                 </tr>
                                 <tr>
                                    <td>Apr 25</td>
                                    <td width="174">Drachman Hall Rm. A120</td>
                                    <td width="437">Dr. Paul Carini, UA, SWES</td>
                                    <td width="555">Understanding the soil microbiome with high throughput microbial cultivation</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </div>
                  </div>
                  <div class="accordion-item">
                     <h2 class="accordion-header" id="headingThree">
                        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
                        Fall 2018
                        </button>
                     </h2>
                     <div id="collapseThree" class="accordion-collapse collapse" aria-labelledby="headingThree" data-bs-parent="#accordionExample">
                        <div class="accordion-body">
                           <table class="table" style="height: 1738px;" width="639">
                              <tbody>
                                 <tr>
                                    <td width="125"><strong>Thursday</strong></td>
                                    <td width="125"><strong>Location</strong></td>
                                    <td width="150"><strong>Speaker</strong></td>
                                    <td width="400"><strong>Title</strong></td>
                                 </tr>
                                 <tr>
                                    <td width="125">August 30</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Scott Saleska</strong>, UA EEB</td>
                                    <td width="400"><em>The promise of ecosystem genomics for advancing our understanding of earth system processes:&nbsp;Four challenges</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">September 6</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Malak Tfaily</strong>,&nbsp;UA SWES</td>
                                    <td width="400"><em>Towards a better understanding of&nbsp;microbial-organic matter interactions&nbsp;and their controls in dynamically changing systems&nbsp;</em>(<a href="https://drive.google.com/open?id=1zXovjKdE48A2L0psX6G6Pcja9PjAQCNE" target="_blank" rel="noopener">Link to video recording</a>)</td>
                                 </tr>
                                 <tr>
                                    <td width="125">September 13</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Bonnie Hurwitz</strong>, UA ABE</td>
                                    <td width="400"><em>Where does the holobiont begin and end? The relationship of host genetics and the microbiome in colon cancer. (</em><a href="https://drive.google.com/open?id=1lZDt8WXH8ZKzLitKskDog491-iFGM42s" target="_blank" rel="noopener">Link to video recording</a><em>)</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">September 20</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Rod Wing</strong>, UA Plant Sciences</td>
                                    <td width="400"><em>Harvesting 15MY of&nbsp;Oryza&nbsp;genome evolution to help solve the 10-billion people question.&nbsp;</em>(<a href="https://drive.google.com/open?id=1nmzvJgR7GeUsiNWAMmK6Gpny8AZhWBtb" target="_blank" rel="noopener">Link to video recording</a>)</td>
                                 </tr>
                                 <tr>
                                    <td width="125">September 27</td>
                                    <td width="125">MRB 102</td>
                                    <td width="150"><strong>Laura Meredith</strong>, UA SNRE</td>
                                    <td width="400"><em>Microbe-mediated trace gas fluxes: an </em><em>ecosystem genomics approach to link genes to atmospheric composition </em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">October
                                       4
                                    </td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Rachel Gallery</strong>, UA SNRE</td>
                                    <td width="400"><em>Predicting microbial activity in soils: How can remote sensing tools help?</em> (<a href="https://drive.google.com/file/d/1H2jauDr3dMcorwa_6h436Diob-vK_Cf1/view?usp=sharing" target="_blank" rel="noopener">Link to video recording</a>)</td>
                                 </tr>
                                 <tr>
                                    <td width="125">October 11</td>
                                    <td width="125">MRB 102</td>
                                    <td width="150"><strong>Susannah Tringe</strong>, Department of Energy, Joint Genome Institute</td>
                                    <td width="400"><em>Linking sequence to biogeochemistry in wetland carbon cycling</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">October 18</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Julie Neilson</strong>, SWES</td>
                                    <td width="400"><em>Soil Microbiome Resilience in Arid Ecosystems</em> (Link to video recording <a href="https://arizona.zoom.us/recording/share/H6mCfsK6UZeem-Ncg5AMyq9JiItJFS10kCAMnima6TWwIumekTziMw" target="_blank" rel="noopener">with</a> and <a href="https://drive.google.com/file/d/1dicKdEKTw1DZE9FCfT7wdlA8vU_BWYHE/view?usp=sharing" target="_blank" rel="noopener">without</a> audio transcript )</td>
                                 </tr>
                                 <tr>
                                    <td width="125">October 25</td>
                                    <td width="125">MRB 102</td>
                                    <td width="150"><strong>Katrina Dlugosch</strong>, UA EEB</td>
                                    <td width="400"><em>Genomics of plants and their associated microbes: evolution of species interactions and its potential role in plant invasions</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">November 1</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Jana U’Ren</strong>, UA ABE</td>
                                    <td width="400"><em>Linking phyllosphere fungal community assembly, evolutionary history, and metabolic diversity to ecosystem function</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">November 8</td>
                                    <td width="125">MRB 102</td>
                                    <td width="150"><strong>Ecosystem Genomics Postdocs</strong></td>
                                    <td width="400"><em>Short presentations by the students and postdocs of the Ecosystem Genomics Initiative</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">November 15</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Russ Monson</strong>, UA Tree Ring Lab</td>
                                    <td width="400"><em>The Semantics of Ecosystem Genomics </em>
                                       &nbsp;
                                    </td>
                                 </tr>
                                 <tr>
                                    <td width="125">November 20, <strong>Tues*</strong></td>
                                    <td width="125">MRB 102</td>
                                    <td width="150"><strong>Duke Pauli</strong>, UA Plant Sciences</td>
                                    <td width="400"><em>Phenomics: Bringing the lab to the field</em></td>
                                 </tr>
                                 <tr>
                                    <td width="125">November 29</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Regis Ferriere</strong>, UA EEB</td>
                                    <td width="400"><em>Modeling microbial evolution to improve our understanding of ecosystem function </em>(<a href="https://drive.google.com/drive/folders/1dDdkGCXD8sArC1PgH1cJcl5ZSinxEIK7?usp=sharing" target="_blank" rel="noopener">Link to video recording</a>)</td>
                                 </tr>
                                 <tr>
                                    <td width="125">December 13</td>
                                    <td width="125">Keating 103</td>
                                    <td width="150"><strong>Brendan Bohannan</strong>, U Oregon</td>
                                    <td width="400"><em>Linking metagenomes to ecosystem functions: what can we learn from genotype-phenotype mapping in organisms? </em>(<a href="https://drive.google.com/file/d/1pRiPsDyts2TN7ucBXSwaZyu527ud8c0c/view?usp=sharing" target="_blank" rel="noopener">Link to video recording</a>)</td>
                                 </tr>
                              </tbody>
                           </table>
                        </div>
                     </div>
                  </div>
               </div>
            </div>
         </div>
      </section>
      <section class="page-section" id="GIDP">
         <div class="container px-4 px-lg-5">
            <h2 class="text-center mt-5">Graduate Programs</h2>
            <hr class="divider"/>
            <div class="row justify-content-center">
               <div class="col-lg-8 text-center">
                  <p class="mb-4">We are connected with these grad programs on campus and have established a new GIDP</p>
               </div>
            </div>
            <div class="row">

               <div class="col-6 text-center mb-5">
                  <a href="https://docs.google.com/spreadsheets/d/1-VA2twRq7mJmt4g5AWKcmni5N89ERqX0/edit#gid=1882979969" target="_blank"><button class="btn btn-primary btn-lg" type="button" data-bs-toggle="collapse" data-bs-target="#collapseExample" aria-expanded="false" aria-controls="collapseExample">
                     Course List
                     </button>
                  </a>
               </div>
               <div class="col-6 text-center mb-5 ">
                  <a href="https://docs.google.com/document/d/1cNwtn3AF4yqt8nKGstgR54T8ZwKPFJ4B/edit" target="_blank"><button class="btn btn-primary btn-lg" type="button" data-bs-toggle="collapse" data-bs-target="#collapseExample" aria-expanded="false" aria-controls="collapseExample">
                     Student Handbook
                     </button>
                  </a>
               </div>
               <div class="col-12">
                  <p>Ecosystem Genomics is an emergent, interdisciplinary area of study that integrates across biotic systems "from genes to ecosystems", with the aim of solving grand challenges in sustainability and innovation in a rapidly changing world. By bringing together ecosystem- and genomic sciences in a framework that connects ecology, sustainability, data science, and applications, ecosystem genomics students have the potential to flourish in diverse careers in industry, government, academia, and non-profit sectors. </p>
                   </div>
               <div class="col-12 mb-5">
                   <p>The ultimate aim of the Ecosystem Genomics GIDP is to foster a diverse new generation of transdisciplinary scientists to address the challenges of sustaining natural and managed ecosystems on which humans depend, including wildlands, agricultural systems, forests, arid lands, and marine environments. The coursework supported by this minor will help students think across scales as they develop skills in ecosystem science, genomics, interdisciplinarity, scientific communication, and collaboration. At is core the minor will foster and extend students’ excellence in areas such as data science, microbiology, plant sciences, insect science, environmental science, atmospheric science, biosystems engineering, ecology and evolutionary biology, geography, and information science, and it is intended to attract students majoring in these UArizona programs. We seek students who want to translate ideas into meaningful scientific advances while cultivating deep and broad skill sets to solve grand challenges relevant to regional, national, and international issues in sustainability and innovation.</p>
               </div>

            </div>
            <div class="row justify-content-center">
               <div class="col-lg-8 text-center">
                  <p class="mb-4">To learn more about the GIDP, please contact Heather Ingram at hci@email.arizona.edu</p>
               </div>
            </div>
      </section>
      <!-- Footer-->
      <footer class="bg-light py-5">
         <div class="container">
            <div class="row">
               <div class="col-12">
            <div class="text-center">
                <img src="assets/img/funding.png" height="150">
            </div>
            <div class="small text-center text-muted mt-2">
               Ecosystem Genomics Initiative, the University of Arizona | © Arizona Board of Regents
               &nbsp;
               <a href="https://www.arizona.edu/" target="_blank" rel="noopener">The University of Arizona</a>
            </div>
            <div class="small text-center text-muted">
            </div>
         </div>
      </div>
      </div>
      </footer>
      <!-- Bootstrap core JS-->
      <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js"></script>
      <!-- SimpleLightbox plugin JS-->
      <script src="https://cdnjs.cloudflare.com/ajax/libs/SimpleLightbox/2.1.0/simpleLightbox.min.js"></script>
      <!-- Core theme JS-->
      <script src="js/scripts.js"></script>
      <!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *-->
      <!-- * *                               SB Forms JS                               * *-->
      <!-- * * Activate your form at https://startbootstrap.com/solution/contact-forms * *-->
      <!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *-->
      <script src="https://cdn.startbootstrap.com/sb-forms-latest.js"></script>
      <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
   </body>
</html>